crates/sui-core/src/consensus_adapter.rs

```diff
@@ -1287,6 +1301,55 @@ impl SubmitToConsensus for Arc<ConsensusAdapter> {
         self.submit_batch(transactions, None, epoch_store)
             .map(|_| ())
     }
+
+    fn submit_best_effort(
+        &self,
+        transaction: &ConsensusTransaction,
+        epoch_store: &Arc<AuthorityPerEpochStore>,
+        // timeout is required, or the spawned task can run forever
+        timeout: Duration,
+    ) -> SuiResult {
+        let permit = match self.submit_semaphore.clone().try_acquire_owned() {
+            Ok(permit) => permit,
+            Err(_) => {
+                return Err(SuiError::TooManyTransactionsPendingConsensus);
+            }
+        };
+
+        let _in_flight_submission_guard =
+            GaugeGuard::acquire(&self.metrics.sequencing_in_flight_submissions);
+
+        let key = SequencedConsensusTransactionKey::External(transaction.key());
+        let tx_type = classify(transaction);
+
+        let async_stage = {
+            let transaction = transaction.clone();
+            let epoch_store = epoch_store.clone();
+            let this = self.clone();
+
+            async move {
+                let _permit = permit; // Hold permit for lifetime of task
+
+                let result = tokio::time::timeout(
```

Comment:
Ignoring the transaction status is ok for this PR. In future consensus may treat the txn as cancelled by Sui when its status receiver has been dropped. We can revisit the logic then.